### ISSUE TITLE
Support for dns name based policy entity rules

### ISSUE THREAD

**Issue by skaegi** [projectcalico/calico#4025]
It's really hard to write reasonable egress rules for out-of-cluster services without using dns names. The IPs for external services can change and as a work-around either a networkpolicy or networkset needs to be updated when this happens. I recognize this is currently a feature of Calico Enterprise -- https://www.tigera.io/features/dns-policy/ -- however it is an incredibly common case and would be good to support directly.

**Comment by ejderdal** [comment]
Agree, this would be very useful. Any news?

**Comment by mrdima** [comment]
Yes, I fully agree that this needs to be an available feature, why would you force people to move to Cilium which can do this without enterprise licensing. Please provide some feedback whether you're considering for the short term or not.

**Comment by Menschomat** [comment]
Any updates on this?

**Comment by tomastigera** [comment]
It is still an enterprise only feature
